max_xy: 20
min_xy: -20

max_z: 8
min_z: 3

max_yaw: 3.14
min_yaw: -3.14

goto_relative_altitude_down: 2.0
goto_relative_altitude_up: 2.0

trajectory:

  p1: -10
  p2: 10

  speed: 2.0
